src/components/chat-list/chat-item/chat-item.tsx

```diff
@@ -131,6 +131,24 @@ const ChatItem: React.FC<IChatItemProps> = React.memo(({ chatId }) => {
     [MessageState.DRAFT]: undefined,
   };
 
+  let lastActivityDate;
```

Comment:
@d1mon1k please create separate memoized function called getMessageDateTime and put this logic there, otherwise this code is executed every time when the component re-renders and the readability will be improved as well

Reply:
I have just done